utility/src/main/java/org/oppia/android/util/networking/BUILD.bazel

```diff
@@ -0,0 +1,17 @@
+"""
+Utilities corresponding to networking utilities.
+"""
+
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "network_connection_util",
+    srcs = [
+        "NetworkConnectionUtil.kt",
+    ],
+    deps = [
+         "//third_party:javax_inject_javax_inject",
+         "//third_party:androidx_annotation_annotation"
```

Comment:
This should move one line up to ensure the dependencies list is lexicographically sorted.

Reply:
Yeah forgot that thanks @BenHenning. I'll do it